mvp-spec/partials-updater/src/interpretor.ts

```diff
@@ -42,13 +50,9 @@ export async function interpret(tokens: LexerToken[]): Promise<string> {
                 if (nextToken.type != LexerTokenType.PartialEnd) {
                     throw new Error(`Partial Begin without Partial End: ${token.text}`);
                 }
-                const lineBreak = getLineBreakSymbol(token.text);
                 doc += token.text;
                 doc += "<!-- ⚠️ GENERATED CONTENT - DO NOT MODIFY DIRECTLY ⚠️ -->" + lineBreak;
-                doc += await interpretPartialBeginToken(token);
-                if (!doc.endsWith(lineBreak)) {
-                    doc += lineBreak;
-                }
```

Comment:
How is it handled now ?

Reply:
Unless I missed something, the case with no line break could only happen if you add the link to an SVG image.
Instead of testing it here, I simply added the missing linebreak here https://github.com/criteo/addressable-network-proposals/pull/50/files/219953002f75fbec7ff7391ce6a6c9848e4c63d4#diff-167b836a7d7856b1281c904f33af8610497fb25ac8f53f4c9a777bd7d3072143R113

Reply:
Well, it can happen if you import a JSON with no linebreak at the end of the file.
But we shouldn't add a line break just in this case. We should just add the "partial end" after a line break, all the time.
Will propose a fix.